showyourwork/workflow/scripts/arxiv.py

```diff
@@ -26,8 +30,22 @@
         f"{ms_name}.fdb_latexmk",
     ]
 
-    # Tar up everything in the src/tex directory
-    with tarfile.open("arxiv.tar.gz", "w:gz") as tarball:
-        for file in paths.user().compile.rglob("*"):
-            if file.name not in exclude and file.suffix != ".bib":
-                tarball.add(file, arcname=file.name)
+    with TemporaryDirectory() as tmpdir:
+        # First, copy to a temporary directory, excluding files
+        shutil.copytree(paths.user().compile, tmpdir, dirs_exist_ok=True)
+
+        if config["preprocess_arxiv"]["enabled"]:
+            # Run the preprocessing script, if provided:
+            script = config["preprocess_arxiv"]["script"]
+            if not script:
```

Comment:
Why do we need the `enabled` config if the real requirement is that the `script` be provided. Why not just have a config called `preprocess_arxiv`, which is just the name of the script?

Reply:
Fine with me!